File: QE-Python Selenium/Activity16.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import Select

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/selects")

print("Page Title:", driver.title)

dropdown_element = driver.find_element(
    By.XPATH, "/html/body/div/main/div/div/div/div/section/div[1]/select"
)

dropdown = Select(dropdown_element)

print("Dropdown options:")
for option in dropdown.options:
    print(option.text)

driver.quit()

File: QE-Java Selenium/Activity16.java
package demos;

import org.openqa.selenium.*;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.Select;


public class Activity16 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/selects");

        System.out.println("Page Title: " + driver.getTitle());

        WebElement selectElem = driver.findElement(
            By.xpath("/html/body/div/main/div/div/div/div/section/div[1]/select")
        );

        Select dropdown = new Select(selectElem);

        System.out.println("Dropdown options:");
        for (WebElement option : dropdown.getOptions()) {
            System.out.println(option.getText());
        }

        driver.quit();
    }
}

